packages/sanity/src/form/members/items/ArrayOfObjectsItem.tsx

```diff
@@ -50,6 +51,25 @@ export function ArrayOfObjectsItem(props: MemberItemProps) {
     }
   })
 
+  const onRemove = useCallback(() => {
```

Comment:
Had to move onRemove up since it is now used by handleClose. 
Also moved onInsert since they belong together imo.